web/html/src/manager/virtualization/nets/properties/NetworkAddress.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+import * as React from 'react';
+import Validation from 'components/validation';
+import { InputBase } from 'components/input/InputBase';
+import { FormContext } from 'components/input/Form';
+import * as utils from './utils';
+
+type Props = {
+  /** name prefix of the field to map in the form model.
```

Comment:
It's very nice that you comment all of these descriptively, these have helped me a lot with migration stuff in other places and are very nice in general. 👍

Reply:
yeah, and those are picked up by storybook docs :)